commit 3e28a96b87f64bc4cbb3a254793fa609df7f2bcc
Author: Hans de Goede <hdegoede@redhat.com>
Date:   Sun Jun 3 15:42:32 2018 +0200

    ASoC: Intel: bytcr_rt5640: Fix Acer Iconia 8 over-current detect threshold
    
    [ Upstream commit f12a0a3c4cc6f594d7c2ea361f2396ae5c518d2c ]
    
    Change the over-current detect threshold on the Acer Iconia 8 from
    2000ua to 1500uA, this fixes headset button presses not being detected.
    
    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Acked-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
    Signed-off-by: Mark Brown <broonie@kernel.org>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/sound/soc/intel/boards/bytcr_rt5640.c b/sound/soc/intel/boards/bytcr_rt5640.c
index 33065ba294a9..d2c9d7865bde 100644
--- a/sound/soc/intel/boards/bytcr_rt5640.c
+++ b/sound/soc/intel/boards/bytcr_rt5640.c
@@ -404,7 +404,7 @@ static const struct dmi_system_id byt_rt5640_quirk_table[] = {
 		},
 		.driver_data = (void *)(BYT_RT5640_DMIC1_MAP |
 					BYT_RT5640_JD_SRC_JD1_IN4P |
-					BYT_RT5640_OVCD_TH_2000UA |
+					BYT_RT5640_OVCD_TH_1500UA |
 					BYT_RT5640_OVCD_SF_0P75 |
 					BYT_RT5640_SSP0_AIF1 |
 					BYT_RT5640_MCLK_EN),
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1198_3e28a96b87f64bc4cbb3a254793fa609df7f2bcc.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1198_3e28a96b87f64bc4cbb3a254793fa609df7f2bcc.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
